src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -31,16 +31,20 @@ If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node
 
 ### Full node {#full-node}
 
-- Stores full blockchain data.
+- Stores full blockchain data (although this is periodically pruned so a full node does not store all state data back to genesis)
 - Participates in block validation, verifies all blocks and states.
-- All states can be derived from a full node.
+- All states can be derived from a full node (although very old states arew reconstructed from requests made to archive nodes).
 - Serves the network and provides data on request.
 
 ### Light node {#light-node}
 
-- Stores the header chain and requests everything else.
-- Can verify the validity of the data against the state roots in the block headers.
-- Useful for low capacity devices, such as embedded devices or mobile phones, which can't afford to store gigabytes of blockchain data.
+Instead of downloading every block, light nodes download block headers. These headers only contain summary information about the contents of the blocks. Any other specific information required by the light node is obtained by requesting it from a full node. The light node can then independently verify the data they receive against the state roots in the block headers. This enables users to participate in the Ethereum network without deploying substantial hardware and bandwidth to running full nodes. Evenutally light nodes could run on mobile phones or embedded devices. The light nodes do not participate in consensus (i.e. they cannot be miners/validators) but they can access the Etheruem blockchain with the same functionality as a full node.
```

Comment:
- Some typos
- Use active voice
- Slight clarity tweak

```suggestion
Instead of downloading every block, light nodes download block headers. These headers only contain summary information about the contents of the blocks. Any other information required by the light node gets requested from a full node. The light node can then independently verify the data they receive against the state roots in the block headers. Light nodes enable users to participate in the Ethereum network without the powerful hardware or high bandwidth required to run full nodes. Eventually, light nodes might run on mobile phones or embedded devices. The light nodes do not participate in consensus (i.e. they cannot be miners/validators), but they can access the Ethereum blockchain with the same functionality as a full node.
```